.github/workflows/actions.yml

```
@@ -6,8 +6,6 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - '2.7.x'
-          - '3.0.x'
           - '3.1.x'
         active_support:
           - 'active_support_6.0.x'
```

```suggestion
          - '3.3.x'
        active_support:
          - 'active_support_7.2.x'
```
maybe keep ruby 3.2
```suggestion
          - '3.2.x'
          - '3.3.x'
        active_support:
          - 'active_support_7.2.x'
```
remove other active_support versions, it only gets security updates
https://endoflife.date/ruby
https://endoflife.date/rails